scripts/build-meteor.sh

```diff
@@ -19,7 +19,7 @@ meteor npm install
 # build the bundle
 printf "\n[-] Building Meteor application...\n\n"
 mkdir -p $APP_BUNDLE_DIR
-meteor build --directory $APP_BUNDLE_DIR
+meteor build --server-only --directory $APP_BUNDLE_DIR
```

Comment:
FYI --server-only flag doesn't prevent client code from being compiled, https://github.com/meteor/meteor/issues/6270
I'm attempting the same thing as @kokokenada and perhaps adding a step in the bash scripts to remove those platforms from the file might get us somewhere.